packages/djangoflow_app/lib/src/app.dart

```diff
@@ -153,17 +153,17 @@ class _AppState extends State<App> with WidgetsBindingObserver {
     super.initState();
     WidgetsBinding.instance.addObserver(this);
     _router = widget.routerBuilder();
-    widget.onStart?.call(context, _router, AppCubit.instance.state);
+    widget.onStart?.call(context, _router, AppCubit().state);
```

Comment:
Thanks for the contribution.

Why should we use 2 different instances of AppCubit in 2 different places, can you please explain?

Reply:
On a second look, another cubit instance was not necessary, I have made an update to that effect